examples/google-cloud-speech/output/google-cloud-speech/lib/google/cloud/speech/v1/doc/google/longrunning/operations.rb

```diff
@@ -0,0 +1,93 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+module Google
+  module Longrunning
```

Comment:
Could someone remind me why we have to put these docs common types in every client?

Reply:
It is so that we can have working links to the types in the docs for our own types. I also thought it would be worth revisiting this decision, so I tried imagining that these common types weren't linked and that I had to google their docs. It was shockingly difficult. I googled: `Google::Protobuf::Duration ruby` and on the first page of results, the only correct result was a link to our [google-cloud-pubsub docs](http://googleapis.github.io/google-cloud-ruby/docs/google-cloud-pubsub/v0.32.1/Google/Protobuf/Duration.html).